src/utils/load-audio.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx createElementEntity */
+import { createElementEntity } from "../utils/jsx-entity";
+import { ProjectionMode } from "./projection-mode";
+import { VideoTexture } from "three";
+import { renderAsEntity } from "../utils/jsx-entity";
+import { loadAudioTexture } from "../utils/load-audio-texture";
+import { HubsWorld } from "../app";
+
+export function* loadAudio(world: HubsWorld, url: string) {
+  const { texture, ratio }: { texture: VideoTexture; ratio: number } = yield loadAudioTexture(url);
+
+  // TODO: VideoTexture.image must be content that be played
+  //       in video-system. And it is also used to render.
+  //       It is audio here so the object will be rendered as
+  //       black. Audio icon must be rendered. Fix this.
+
+  return renderAsEntity(
+    world,
+    <entity
+      name="Audio"
+      networked
+      networkedVideo
+      grabbable={{ cursor: true, hand: false }}
+      // Audio and Video are handled very similarly in 3D scene
+      // so create as video
+      video={{
```

Comment:
Since we want to change the visual representation it might be nicer to have an audio inflator. It could end up using the MediaVideo component if it needs to, but it could create a different mesh with a static image.

Reply:
> it could create a different mesh with a static image.

If I'm right, currently the video system is designed that `object.material.map` is used both for rendering and playing media so it's impossible to render a static image for audio. I didn't want to add workaround in the video system or write a separated audio system (because most of the processing would be like the same), and didn't have time to refactor the design. I want to think of properer design/implementation in the next cycle.